starter-files/date.js

```diff
@@ -48,13 +48,22 @@ export function dateToString(date) {
     return null;
   const d = new Date(date);
   const ampm = d.getHours() > 12 ? 'PM' : 'AM';
-  return `${
-    MONTH_NAMES[d.getMonth()]
-  } ${d.getDate()}, ${d.getFullYear()} ${padLeadingZeroes(
-    d.getHours() % 12,
-    2
-  )}:${padLeadingZeroes(
-    d.getMinutes(),
-    2
-  )}:${padLeadingZeroes(d.getSeconds(), 2)} ${ampm}`;
+  if (d.getHours() === 12) {
```

Comment:
I think we may need your fix for `13:00`, `14:00`, etc...

Let's combine this with the `AM`/`PM` condition. Something like

```js
let hrs = d.getHours();
let ampm = 'AM';
if (hrs > 12) {
  // afternoon
  ampm = 'PM';
  hrs = hrs % 12;
}
```

And then we don't need another condition to switch between using `d.getHours()` and `d.getHours() % 12` (`hrs` will already be what we want)

Reply:
I think this will leave everything in military time, but I'm not sure that's what you wanted?

Reply:
updated the date.js util after working on it a little more.